src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,61 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import React from 'react';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  setCurrentPage: (value: number) => void;
+  itemsPerPage: number;
+  currentPage: number;
+};
+
+export const Pagination: React.FC<Props> = ({
+  itemsPerPage,
+  setCurrentPage,
+  currentPage,
+}) => {
+  const lastPage = Math.ceil(42 / itemsPerPage);
+  const pages = getNumbers(1, lastPage);
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', { disabled: currentPage === 1 })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
```

Comment:
You check if currentPage === 1 not once. Move it to a variable to not repeat the same code